pages/common/nginx.md

```diff
@@ -8,4 +8,4 @@
 
 - Start server with custom config and custom relative path base directory
 
-`nginx -c {{path to config}} -p {{relative path base dir}}`
+`nginx -c {{config_file}} -p {{relative/path/to/base/dir}}`
```

Comment:
Maybe even better `nginx -c {{config_file}} -p {{/path/to/resolve/relative/paths/against}}`. WDYT?